data/json/recipes/recipe_food.json

```diff
@@ -9100,6 +9100,36 @@
     ]
   ]
 },{
+   "type" : "recipe",
+   "result": "quesadilla_cheese",
+   "category": "CC_FOOD",
+   "subcategory": "CSC_FOOD_SNACK",
+   "skill_used": "cooking",
+   "difficulty": 0,
+   "time": 5000,
+   "reversible": false,
+   "autolearn": true,
+   "qualities" : [
+     {"id":"COOK","level":1,"amount":1}
+  ], "tools": [
+   [
+      [ "hotplate", 3 ],
+      [ "char_smoker", 1 ],
+      [ "toolset", 3 ],
+      [ "fire", -1 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "cheese", 1 ],
+      [ "can_cheese", 1],
+      [ "cheese_hard", 1]
+    ],
+    [
+      [ "tortilla_corn", 2 ]
```

Comment:
@chaosvolt The code is the way it is because of the method used to create it, that's pretty inevitable. But that method is also how the game got to be as awesome as it is. This is the best roguelike I've ever played and I love it. Don't be a debbie downer, if you aren't happy with something, try to make it better. Try playing original cataclysm for an hour and tell me things aren't much much better than they were. Complaining just gives others a bad impression.